apostrophecms/form
# Feature/select multiple

## Summary

Add multiple and size option to select field widget

## What are the specific steps to test this change?

1. Add a select field widget to a form
2. In the widget editor modal, set `Allow multiple options to be selected` to `Yes`
3. Set the `size` to the value you want.
4. Check the rendered template in Apostrophe admin UI and on the website

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
